admin-dev/themes/default/template/controllers/cart_rules/form.tpl

```diff
@@ -78,6 +78,9 @@
 
     {if isset($refresh_cart) }
       if (typeof window.parent.order_create !== "undefined") {
+		  {if isset($created_cart_rule) }
```

Comment:
Can you add a comment on this whole block (before the `if isset($refresh_cart)`) to explain the purpose of this code It is related to the "Create order page" which can create cart rule via a fancy box and needs to update its content via JS code calls

Reply:
This will be important when we migrate this page, to understand why this piece of code is present here and needs to be migrated or, even better, modified with a cleaner solution (dedicated form, or smarter JS code that is able to fetch the create CartRuleId)

Reply:
As suggested in the cart controller, these two functions call and two variables could be merged into one because it's the same use case/behaviour